sdk/monitor/Microsoft.OpenTelemetry.Exporter.AzureMonitor/tests/Microsoft.OpenTelemetry.Exporter.AzureMonitor.Integration.Tests/FunctionalTests/AzureMonitorTestEnvironment.cs

```diff
@@ -0,0 +1,27 @@
+﻿// Copyright (c) Microsoft Corporation. All rights reserved.
+// Licensed under the MIT License.
+
+namespace Microsoft.OpenTelemetry.Exporter.AzureMonitor.Integration.Tests.FunctionalTests
+{
+    using global::Azure.Core.TestFramework;
+
+    public class AzureMonitorTestEnvironment : TestEnvironment
+    {
+        public AzureMonitorTestEnvironment() : base()
+        {
+        }
+
+        public string ConnectionString => GetRecordedVariable(EnvironmentVariableNames.ConnectionString);
+
+        public string InstrumentationKey => GetRecordedVariable(EnvironmentVariableNames.InstrumentationKey);
```

Comment:
My gut feeling is that we don't want to use `InstrumentationKey` anymore, and we would stick with ConnectionString. So keeping both could be confusing?